snippets/Google.Storage.V1.Snippets/StorageClientSnippets.cs

```diff
@@ -126,12 +126,12 @@ public void ListObjects()
         }
 
         [Fact]
-        public void DownloadFile()
+        public void DownloadObject()
```

Comment:
Maybe - on the other hand, if I'm trying to find an example of the DownloadObject call, looking through the list of methods is the simple way of doing it. I don't feel strongly about it though - I'm happy to change them back if you want.